graphrag_sdk/ontology.py

```
@@ -103,12 +104,84 @@ def from_graph(graph: Graph):
+        Args:
+            attributes (list[list[list[str]]]): The attributes extracted from the graph.
+
+        Returns:
+            processed_attributes (list[Attribute]): The processed attributes.
+        """
+        processed_attributes = []
+        for attr in attributes:
+            attr_name, attr_type = attr[0]
+            try:
+                attr_type = AttributeType.from_string(attr_type)
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)